.buildconfig/macos-latest.yml

```diff
@@ -0,0 +1,25 @@
+name: macos-latest
+channels:
+  - conda-forge
+dependencies:
+  - cmake
+  - conan>=1.45.0
+  - confuse
+  - graphlib-backport
+  - h5py
+  - hypothesis
+  - ipywidgets
```

Comment:
I think you don't need `ipywidgets` if you require `pythreejs` below (one less line in the dependency list)

Reply:
Aren't we using ipywidgets for all the GUI elements of normal plots? We don't want that to break if we switch to a different library for scatter plots?